images/hook/entrypoint.sh

```diff
@@ -15,6 +15,17 @@ if [ $1 = "update" ]; then
         sed -i s/b1lIV3gyVDlmQVd3SzdsZTRrZDY=/cm9vdA==/g /var/lib/gravity/resources/influxdb-secret.yaml
         rig upsert -f /var/lib/gravity/resources/influxdb-secret.yaml --debug
     fi
+
+    # Drop series containing 'sys_container' type which are not used anywhere and filling up the database.
+    # Skip this step if infuxdb is not ready. Cleanup is not important to fail the upgrade.
+    if kubectl --namespace=monitoring wait --for condition=Available deployments influxdb --timeout=60s >/dev/null 2>&1
```

Comment:
Just to double-check, can the failure of this kubectl command, or any of the evaluated commands below (`$(kubectl ...)`) lead to the hook failure? Just wanna make sure we're not introducing additional failure scenarios...

Reply:
Yes, it absolutely fails aware. If `if` fails the whole block is skipped. When something inside `if` block failed, then `|| true` handles this failure.